### ISSUE TITLE
Api change

### ISSUE THREAD

**Issue by mMdooD** [YarmolenkoD/notpixel#116]
Hello i just want to say an idea that the bot can detect the api change and shut down, please add this possibility and tnxx

**Comment by binchanhkun99** [comment]
> Xin chào, tôi chỉ muốn nói một ý tưởng rằng nếu api bot thay đổi, tập lệnh có thể phát hiện ra sự thay đổi api và tắt, vui lòng thêm khả năng này và cảm ơn

Good idea, However, if the api is changed, it is possible that the api has called before being detected....
